app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -55,7 +55,7 @@
 // $rem-base: 16px;
 
 // Allows the use of rem-calc() or lower-bound() in your settings
-@import "foundation/functions";
+@import 'foundation/functions';
```

Comment:
@SamLau95 not a huge problem, but when i run scss-lint on local it prefers single-quoted strings!  i'm using default configs each time -- does hound have slightly different defaults?